frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/EditMyInfoChildren/EditMyInfoChildren.tsx

```diff
@@ -63,6 +64,14 @@ export const EditMyInfoChildren = ({
   return (
     <CreatePageDrawerContentContainer>
       <VStack align="flex-start">
+        <VStack>
+          <Box pt="2rem" pb="1.5rem">
```

Comment:
Looks like there's an additional top padding here (looks like we're leaving too much space before the `<Box />`).